winlogbeat/_meta/fields.common.yml

```diff
@@ -60,23 +72,33 @@
       description: >
         The keywords are used to classify an event.
 
+    - name: channel
+
+    - name: provider_name
```

Comment:
Just adding the two windows event log fields that were initially mapped like this:

- channel was `log_name`. Example values:
  - Application
  - Security
  - Setup
- provider_name was `source_name`. Example values:
  - Microsoft-Windows-Security-Auditing
  - Service Control Manager
  - Microsoft-Windows-CEIP

I think the Channel idea may be interesting to map to ECS (e.g. is there a correlation with syslog's facility?), but I'm not sure for the provider name. It's interesting data, but I'm not even sure I would map that to event.origin, which has come up every once in a while. Needs more thought (aka not for FF).